### PR TITLE
cleanups: Kotlin Cleanup & mLongClickTestRunnable

### PR DESCRIPTION
**Pull request by david-allison**
## Purpose / Description
* I came across a confusing `KotlinCleanup` and want to ensure others don't spend time on it

Afterwards, I investigated a use of `vibrate` and found that it was never called

## How Has This Been Tested?
⚠️ Untested

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/compat/CompatV26.kt [hunk 1]
```diff
@@ -44,11 +43,9 @@ open class CompatV26 : CompatV24(), Compat {
     override fun setTooltipTextByContentDescription(view: View) { /* Nothing to do API26+ */
     }
 
-    @Suppress("DEPRECATION")
-    @KotlinCleanup("when solving the deprecation of Context.VIBRATOR_SERVICE fix the SENSELESS_COMPARISON warning")
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/compat/CompatV26.kt:48]: @lukstbit I believe this was already handled in https://github.com/ankidroid/Anki-Android/commit/0e4347289197b101ce84586a836669d23a86d42d#diff-911a74b76496404a57304ae9a6d4873bdc6a75792754e413730b4b7470d8d615

If you remember, could I confirm I'm not missing something

Comment (lukstbit) [reply AnkiDroid/src/main/java/com/ichi2/compat/CompatV26.kt:48]: Yes, it was already handled and can be deleted! I should have handled the _senseless_comparison_ when I migrated the class.

**Review by lukstbit — APPROVED** [review]
LGTM
The Compat.vibrate() method became unused. Do we keep it for (maybe) future work?

**Comment by david-allison** [comment]
> The Compat.vibrate() method became unused. Do we keep it for (maybe) future work?

Yes, it's going to be used in #14591

**Review by BrayanDSO — APPROVED** [review]

**Pull request merged**